### PR TITLE
Add a command to tell people what to do if they have custom requests

### PR DESCRIPTION
**Pull request by carc1n0gen**
Totally willing to accept suggestions on wording change btw

### CODE REVIEW

File: commands.json [hunk 1]
```diff
@@ -38,5 +38,18 @@
                 "value": "The most common brickless configuration is using an HDPLEX 400W AC-DC module along the side of the chassis near the GPU to convert mains power input to 19V DC output. This is then fed to an HDPLEX 400W HiFi DC-ATX module, which converts the 19V to the various voltages and connectors necessary for the motherboard, GPU, and SATA drives (if used)."
             }
         ]
+    },
+
+    "!custom": {
+        "info": "!custom\t\tHow to get custom s4m work done",
```
Comment (mikedalton) [inline commands.json:44]:
```suggestion
        "info": "How to get custom s4m work done",
```
Not using the command name in the info field now.

File: commands.json [hunk 2]
```diff
@@ -38,5 +38,18 @@
...
+        "color": "#0099ff",
+        "title": "Custom Requests",
+        "description": "",
+        "fields": [
```
Comment (mikedalton) [inline commands.json:48]:
```suggestion
        "dm": false
        "fields": [
```
Using a "dm" field now (true/false) to handle content too long for the main channel.

**Comment by mikedalton** [comment]
Might want to update from master, since I changed the data structure a bit.

**Comment by carc1n0gen** [comment]
> 
> 
> Might want to update from master, since I changed the data structure a bit.

Github let's me auto-update with your changes right here, so it should be good now.

**Pull request merged**